<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberEdit</class>
 <widget class="QDialog" name="memberEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="firstLabel">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="firstInput"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lastLabel">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lastInput"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="userLabel">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="userInput"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="passwordInput"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="gradeLabel">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Grade</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="gradeDropDown">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Freshman</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sophomore</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Junior</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Senior</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="submitButton">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="removeButton">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
